packages/react/src/components/Tooltip/next/Tooltip.stories.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import './story.scss';
+
+import { Checkbox16 } from '@carbon/icons-react';
+import React from 'react';
+import { Tooltip } from '../next';
+
+export default {
+  title: 'Experimental/unstable_Tooltip',
+  component: Tooltip,
+  parameters: {
+    controls: {
+      hideNoControlsWarning: true,
+    },
+    layout: 'centered',
+  },
+  argTypes: {
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+    className: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+};
+
+export const Default = () => {
+  const label =
+    'Occassionally, services are updated in a specified time window to ensure no down time for customers.';
+  return (
+    <Tooltip align="bottom" defaultOpen label={label}>
+      <button className="demo-tooltip-trigger" type="button">
+        <Checkbox16 />
+      </button>
+    </Tooltip>
+  );
+};
+
+export const Duration = () => {
+  return (
+    <>
+      <Tooltip label="Label one" enterDelayMs={500} leaveDelayMs={500}>
+        <button className="demo-tooltip-trigger" type="button">
+          <Checkbox16 />
+        </button>
+      </Tooltip>
+      <Tooltip label="Label two" enterDelayMs={500} leaveDelayMs={500}>
+        <button className="demo-tooltip-trigger" type="button">
+          <Checkbox16 />
+        </button>
+      </Tooltip>
+      <Tooltip label="Label two" enterDelayMs={500} leaveDelayMs={500}>
+        <button className="demo-tooltip-trigger" type="button">
+          <Checkbox16 />
+        </button>
+      </Tooltip>
+    </>
+  );
+};
+
+const PlaygroundStory = (props) => {
+  const { align, defaultOpen, description, label } = props;
```

Comment:
I think this needs a `...rest` to then be applied to the Tooltip below. The `enterDelayMs` and `leaveDelayMs` controls aren't working for me.

https://user-images.githubusercontent.com/3360588/140809133-66c35575-1bb5-49bf-bdaa-9e9f6d5706b8.mov